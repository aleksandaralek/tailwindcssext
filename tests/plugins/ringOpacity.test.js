import { quickPluginTest } from '../util/run'

quickPluginTest('ringOpacity', {
  safelist: [
    // Arbitrary values
    'ring-opacity-[0.12]',
    'ring-opacity-[var(--my-value)]',
  ],
}).toMatchSnapshot()

// With `respectDefaultRingColorOpacity` enabled
quickPluginTest('ringOpacity', {
  safelist: [
    // Arbitrary values
    'ring-opacity-[0.12]',
    'ring-opacity-[var(--my-value)]',
  ],
  future: {
    respectDefaultRingColorOpacity: true,
  },
}).toMatchSnapshot()
