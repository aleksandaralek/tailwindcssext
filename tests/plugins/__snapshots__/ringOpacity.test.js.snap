// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the 'ringOpacity' plugin 1`] = `
"
.ring-opacity-0 {
  --tw-ring-opacity: 0;
}

.ring-opacity-10 {
  --tw-ring-opacity: .1;
}

.ring-opacity-100 {
  --tw-ring-opacity: 1;
}

.ring-opacity-15 {
  --tw-ring-opacity: .15;
}

.ring-opacity-20 {
  --tw-ring-opacity: .2;
}

.ring-opacity-25 {
  --tw-ring-opacity: .25;
}

.ring-opacity-30 {
  --tw-ring-opacity: .3;
}

.ring-opacity-35 {
  --tw-ring-opacity: .35;
}

.ring-opacity-40 {
  --tw-ring-opacity: .4;
}

.ring-opacity-45 {
  --tw-ring-opacity: .45;
}

.ring-opacity-5 {
  --tw-ring-opacity: .05;
}

.ring-opacity-50 {
  --tw-ring-opacity: .5;
}

.ring-opacity-55 {
  --tw-ring-opacity: .55;
}

.ring-opacity-60 {
  --tw-ring-opacity: .6;
}

.ring-opacity-65 {
  --tw-ring-opacity: .65;
}

.ring-opacity-70 {
  --tw-ring-opacity: .7;
}

.ring-opacity-75 {
  --tw-ring-opacity: .75;
}

.ring-opacity-80 {
  --tw-ring-opacity: .8;
}

.ring-opacity-85 {
  --tw-ring-opacity: .85;
}

.ring-opacity-90 {
  --tw-ring-opacity: .9;
}

.ring-opacity-95 {
  --tw-ring-opacity: .95;
}

.ring-opacity-\\[0\\.12\\] {
  --tw-ring-opacity: .12;
}

.ring-opacity-\\[var\\(--my-value\\)\\] {
  --tw-ring-opacity: var(--my-value);
}
"
`;

exports[`should test the 'ringOpacity' plugin 2`] = `
"
.ring-opacity {
  --tw-ring-opacity: .5;
}

.ring-opacity-0 {
  --tw-ring-opacity: 0;
}

.ring-opacity-10 {
  --tw-ring-opacity: .1;
}

.ring-opacity-100 {
  --tw-ring-opacity: 1;
}

.ring-opacity-15 {
  --tw-ring-opacity: .15;
}

.ring-opacity-20 {
  --tw-ring-opacity: .2;
}

.ring-opacity-25 {
  --tw-ring-opacity: .25;
}

.ring-opacity-30 {
  --tw-ring-opacity: .3;
}

.ring-opacity-35 {
  --tw-ring-opacity: .35;
}

.ring-opacity-40 {
  --tw-ring-opacity: .4;
}

.ring-opacity-45 {
  --tw-ring-opacity: .45;
}

.ring-opacity-5 {
  --tw-ring-opacity: .05;
}

.ring-opacity-50 {
  --tw-ring-opacity: .5;
}

.ring-opacity-55 {
  --tw-ring-opacity: .55;
}

.ring-opacity-60 {
  --tw-ring-opacity: .6;
}

.ring-opacity-65 {
  --tw-ring-opacity: .65;
}

.ring-opacity-70 {
  --tw-ring-opacity: .7;
}

.ring-opacity-75 {
  --tw-ring-opacity: .75;
}

.ring-opacity-80 {
  --tw-ring-opacity: .8;
}

.ring-opacity-85 {
  --tw-ring-opacity: .85;
}

.ring-opacity-90 {
  --tw-ring-opacity: .9;
}

.ring-opacity-95 {
  --tw-ring-opacity: .95;
}

.ring-opacity-\\[0\\.12\\] {
  --tw-ring-opacity: .12;
}

.ring-opacity-\\[var\\(--my-value\\)\\] {
  --tw-ring-opacity: var(--my-value);
}
"
`;
